README.md

```diff
@@ -2,21 +2,63 @@
 
 # prom-label-proxy
 
-The prom-label-proxy can enforce a given label in a given PromQL query, in Prometheus API responses or in Alertmanager API requests.
+The prom-label-proxy can enforce a given label in a given PromQL query, in Prometheus API responses or in Alertmanager API requests. This allows read multi-tenancy for projects like Prometheus, Alertmanager or Thanos.
```

Comment:
Multi tenancy is intentionally not mentioned in this section, because this component is more generic than that.

Reply:
I see, what about saying `For example (but not only)`

Reply:
What about now?